specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/examples/Patches_Apply.json

```diff
@@ -0,0 +1,88 @@
+{
+  "operationId": "Patches_Apply",
+  "title": "Patches_Apply_0",
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "builderName": "testBuilder",
+    "patchName": "testPatch-25fe4b",
+    "api-version": "2023-08-01-preview"
```

Comment:
>"api-version": "2023-08-01-preview" [](http://example.com/codeflow?start=4&length=35)

So now I'm curious, aren't any other parameters required? How did it know which patches to apply?